github/WorkflowJob.py

```diff
@@ -0,0 +1,201 @@
+    """
+    This class represents Workflow Jobs. The reference can be found here https://docs.github.com/en/rest/reference/actions#workflow-jobs
+    """
+
+    def __repr__(self):
+        return self.get__repr__({"id": self._id.value, "url": self._url.value})
+
+    @property
+    def check_run_url(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._check_run_url)
+        return self._check_run_url.value
+
+    @property
+    def completed_at(self):
+        """
+        :type: datetime.datetime
+        """
+        self._completeIfNotSet(self._completed_at)
+        return self._completed_at.value
+
+    @property
+    def conclusion(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._conclusion)
+        return self._conclusion.value
+
+    @property
+    def head_sha(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._head_sha)
+        return self._head_sha.value
+
+    @property
+    def html_url(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._html_url)
+        return self._html_url.value
+
+    @property
+    def id(self):
+        """
+        :type: int
+        """
+        self._completeIfNotSet(self._id)
+        return self._id.value
+
+    @property
+    def name(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._name)
+        return self._name.value
+
+    @property
+    def node_id(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._node_id)
+        return self._node_id.value
+
+    @property
+    def run_id(self):
+        """
+        :type: integer
+        """
+        self._completeIfNotSet(self._run_id)
+        return self._run_id.value
+
+    @property
+    def run_url(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._run_url)
+        return self._run_url.value
+
+    @property
+    def started_at(self):
+        """
+        :type: datetime.datetime
+        """
+        self._completeIfNotSet(self._started_at)
+        return self._started_at.value
+
+    @property
+    def status(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._status)
+        return self._status.value
+
+    @property
+    def steps(self):
+        """
+        :type: list of github.WorkflowStep.WorkflowStep
+        """
+        self._completeIfNotSet(self._steps)
+        return self._steps.value
+
+    @property
+    def url(self):
+        """
+        :type: string
+        """
+        self._completeIfNotSet(self._url)
+        return self._url.value
+
+    def logs_url(self):
+        """
+        :type: string
+        """
+        print(f"{self.url}/logs")
+        headers, _ = self._requester.requestBlobAndCheck("GET", f"{self.url}/logs")
+        return headers["location"]
```

Comment:
Let's get this in line with `WorkflowRun`:
```suggestion
    @property 
    def logs_url(self): 
        """ 
        :type: string 
        """ 
        self._completeIfNotSet(self._logs_url) 
        return self._logs_url.value 
```